Assets/Plugins/Editor/Uplift/Testing/DependencyResolution/TestDependencyResolution.cs

```diff
@@ -0,0 +1,932 @@
+using Uplift.Common;
+using Uplift.Packages;
+using Uplift.Schemas;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine.TestTools;
+using UnityEngine;
+using System.Text;
+using System.Linq;
+
+namespace Uplift.DependencyResolution
+{
+	[TestFixture]
+	public class TestDependencyResolution
```

Comment:
Weird indentation throughout the whole file in `Upset[] expected = {...}` declarations